sql/core/src/test/scala/org/apache/spark/sql/execution/BenchmarkWholeStageCodegen.scala

```diff
@@ -59,6 +59,55 @@ class BenchmarkWholeStageCodegen extends SparkFunSuite {
     benchmark.run()
   }
 
+  def testStddev(values: Int): Unit = {
```

Comment:
The test actually touches both `stddev` and `kurtosis`, which should be reflected by the name.